src/components/shared/Input/TokenInput.vue

```diff
@@ -359,11 +367,20 @@ $el-input-class: '.el-input';
     justify-content: space-between;
   }
 
-  .delimiter {
+  // overwrite
+  .el-slider__runway.disabled {
+    .el-slider__bar {
+      width: 0 !important;
```

Comment:
the same comment - if we need to apply anything we cannot do it scoped scss then we need to replace it